docs/team/junwei3112.md

```diff
@@ -10,8 +10,46 @@ The user interacts with it using a CLI, and it has a GUI created with JavaFX. It
 
 Given below are my contributions to the project:
 
-* **Documentation**:
-  * User Guide:
-    * Added documentation for the feature `list`
-  * Developer Guide:
+* **Code contributed**:
+  * My contributions to the code base can be found [here](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=W16-3&sort=groupTitle&sortWithin=title&since=2021-09-17&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabAuthor=JunWei3112&tabRepo=AY2122S1-CS2103T-W16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&tabType=authorship).
```

Comment:
You can consider linking the [report](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=JunWei3112&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17) that displays just your own contributions

Reply:
I have changed the link, thanks!